src/main/java/compal/logic/commands/RemindCommand.java

```diff
@@ -0,0 +1,71 @@
+package compal.logic.commands;
+
+import compal.compal.Compal;
+import compal.logic.parser.CommandParser;
+import compal.tasks.TaskList;
+
+import java.text.ParseException;
+import java.util.Date;
+import java.util.Scanner;
+
+import static compal.compal.Messages.MESSAGE_INVALID_TASK_NUMBER;
+import static compal.compal.Messages.MESSAGE_INVALID_TASK;
+import static compal.compal.Messages.MESSAGE_MISSING_COMMAND_ARG;
+
+/**
+ * Executes user command "remind".
+ */
+public class RemindCommand extends Command implements CommandParser {
+
+    private TaskList taskList;
+
+    /**
+     * Constructs RemindCommand object.
+     *
+     * @param d Compal
+     */
+    public RemindCommand(Compal d) {
+        super(d);
+        this.taskList = d.tasklist;
+    }
+
+    /**
+     * Marks task as has reminder based on user task number input and
+     *      * prints confirmation message to user.
+     *
+     * @param userIn Entire user input string.
+     * @throws Compal.DukeException If user task number input is invalid or missing.
+     * @throws ParseException If date is in invalid format.
+     */
+    @Override
+    public void parseCommand(String userIn) throws Compal.DukeException, ParseException {
+        Scanner scanner = new Scanner(userIn);
+        scanner.next();
+        int maxSize = taskList.arrlist.size();
+        Date currentDate = java.util.Calendar.getInstance().getTime();
+        if(scanner.hasNextLine()) {
```

Comment:
'if' is not followed by whitespace
```suggestion
        if (scanner.hasNextLine()) {
```